src/index.tsx

```diff
@@ -48,6 +49,7 @@ const AuthenticatedApp = () => {
   return (
     (authStateLoaded && configStateLoaded && auth.isAuthenticated && (
       <Apollo>
+        <SandboxFeature<{}> feature={`${config.partnerCompany}/CustomStyle`} />
```

Comment:
Thought this way it would be more future-proof than hardcoding the company name like `feature="FintrafficCustomStyle"`